oktinaut/babylonjs-typescript-starter
# Upgrade Babylon.js from v4 to v5 + Add inspector + Webpack update

+ Update BabylonJS to latest
+ Update webpack to latest
+ Add inspector as a dev dependency (allows the inspector to be used)
+ Show inspector on index.ts

The current project has 11 vulnerabilities (2 moderate, 9 high) according to NPM.

This PR will bring this to **0 vulnerabilities**

I decided to update all to the latest to avoid these issues + add the ability to use inspector which is not straightforward with Typescript. 

Tested both Dev server and Build and both report no issues in console and run as expected.
